spec/request/gi/institutions_request_spec.rb

```diff
@@ -1,4 +1,4 @@
-# frozen_string_literal: true
+# frozen_string_literal: false
```

Comment:
Total NITPICK, but why not just copy and paste the odd character instead of using force_encoding? That way you don't need to modify any of the frozen_string_literals.